### PR TITLE
Pipe weapons easier to craft

### PR DESCRIPTION
**Pull request by Jeff-jefferson-jeffery**
Makes pipe weapons easier to craft by reducing the skill requirement to fabrication 2 mechanics 1 for all basic pipe guns and the single barrel shotgun, and fabrication 3 mechanics 1 for the double barrel.
All pipe weapons no longer require fine sawing, only regular metal sawing, so a multitool can be used to craft them.
This is intended to make them much easier to create at the point in the game where they are useful (the very beginning where you may have loose ammo due to a zombie drop but not a weapon to use it in)

### CODE REVIEW

**Comment by Zireael07** [comment]
IIRC at some point we had makeshift pipe weapons - let's reintroduce them instead of making them unrealistically easy (fine sawing is necessary for the barrel to be properly made IIUC)

**Comment by Jeff-jefferson-jeffery** [comment]
I don't see the point to doing that - the weapons made easier to craft in this PR are the absolute lowest of the low, guns are pretty common right now so by the time you have access to a hacksaw and screwdriver set, you also probably have a real gun, and every real gun is better than any of these.
"better" pipe guns already exist - they haven't had their recipes change and still require precision tools to craft.

File: data/json/recipes/recipe_weapon.json [hunk 1]
```diff
@@ -637,7 +637,8 @@
       ]
     ],
     "components": [
-      [ [ "shotgun_s", 1 ], [ "pipe_shotgun", 1 ] ],
+      [ [ "shotgun_s", 1 ], [ "
+                             _shotgun", 1 ] ],
```
Comment (Coolthulhu) [inline data/json/recipes/recipe_weapon.json:641]: This doesn't look right

Comment (Jeff-jefferson-jeffery) [reply data/json/recipes/recipe_weapon.json]: should be fixed now check again

**Comment by chaosvolt** [comment]
Per discussion on the modder's discord, what's basically the issue with pipe guns at present: the crafting requirement of mechanics 3 (and to a lesser extent being restricted to a hacksaw) is a major detriment to having pipe guns actually be obtainable before you get your hands on a gun and ammo.

Proposed changes would thus make pipe rifles potentially accessible at a point where the player doesn't yet have a decent firearm, which is exactly when pipe rifles are useful. A future PR that makes ammunition more accessible than guns might also compliment this change, by making it more likely that the player will actually run into this situation in the early game. For now though, the idea behind the change basically amounts to making pipe rifles more accessible in the situation where they're most effective.

File: data/json/recipes/recipe_weapon.json [hunk 2]
```diff
@@ -870,13 +870,13 @@
...
-    "skills_required": [ "gun", 1 ],
-    "difficulty": 3,
+    "skill_used": "fabrication",
+    "skills_required": [ "mechanics", 1 ],
+    "difficulty": 2,
     "time": "14 m",
     "reversible": true,
     "autolearn": true,
-    "qualities": [ { "id": "SAW_M_FINE", "level": 1 }, { "id": "SCREW_FINE", "level": 1 } ],
+    "qualities": [ { "id": "SAW_ME", "level": 1 }, { "id": "SCREW_FINE", "level": 1 } ],
```
Comment (Coolthulhu) [inline data/json/recipes/recipe_weapon.json:879]: Typo here

**Comment by Jeff-jefferson-jeffery** [comment]
gonna close this because i have no idea how to fix the typo - my changes aren't going through, i think it's something to do with the de-linking
i will remake it when i'm more git literate

**Pull request closed without merging**